### PR TITLE
Swerve2022: Swerve Making Consts for Drivetrain

### PR DESCRIPTION
**Pull request by hpark-24**

### CODE REVIEW

**Review by amull23 — APPROVED** [review]
looks good!

File: Swerve2022/src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -3,20 +3,34 @@
...
 {
-    /**
-    * public static final class DriveConstants {
-    *   public static final int kLeftMotor1Port = 0;
-    *   public static final int kLeftMotor2Port = 1;
-    *   public static final int kRightMotor1Port = 2;
-    *   public static final int kRightMotor2Port = 3; 
-    * }
-    */
+  /**
```
Comment (jmullins16) [inline Swerve2022/src/main/java/frc/robot/Constants.java:15]: Let's take out the example constants for Drivetrain, since you've added better ones below.

**Review by jmullins16 — APPROVED** [review]
Approving, but take out the example comments next time you make a change. We will need to renumber the CAN IDs on the actual robot to match these constants.

Looks good and you've got the constants loaded in properly. Can be merged as is. (hint, hint to the rest of CSP in reviewing and approving this simple change ASAP.)

Next step is to go inside of SwerveModule.java and change the PWMSparkMax to be WPI_TalonFX objects. This might cause some methods referencing the drive objects to change, but probably won't. See if this will compile and work from there.

**Review by reevar24 — APPROVED** [review]
Great start on the Swerve project!

**Pull request merged**